pages/glossary.md

```diff
@@ -0,0 +1,157 @@
+
+## Account Abstraction
+
+Account Abstraction is the new paradigm that focuses on improving the user experience when interacting with the blockchain by replacing the use of [externally-owned accounts](#externally-owned-account) with [smart accounts](#smart-account).
+
+Some common features that Account Abstraction can fix are:
+- Elimination of seed phrase reliance
+- Ease of multichain interactions
+- Account recovery
+- [Gasless transactions](#gasless-transaction)
+- Transaction batching
+
+See also:
+- [Account Abstraction](https://ethereum.org/en/roadmap/account-abstraction) on ethereum.org
+- [EIP-4337: Account Abstraction](https://www.erc4337.io) on erc4337.io
+
+## EIP-1271
+
+The [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271) is an [Ethereum Improvement Proposal](https://eips.ethereum.org) that proposes a standard way for any contract to verify whether a signature on behalf of a given contract is valid. This is possible via the implementation of a `isValidSignature(hash, signature)` function on the signing contract, which can be called to validate a signature.
+
+## EIP-712
+
+The [EIP-712](https://eips.ethereum.org/EIPS/eip-712) is an Ethereum Improvement Proposal that proposes a standard for hashing and signing of typed structured data as opposed to just bytestrings.
+
+## ERC-4337
+
+The [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) is an [Ethereum Request for Comments](https://eips.ethereum.org/erc) that introduces a higher-layer pseudo-transaction object called `UserOperation`. Users send `UserOperation` objects into a separate mempool. A special class of actor called bundlers package up a set of these objects into a transaction making a `handleOps` call to a special contract, and that transaction then gets included in a block.
+
+See also:
+- [ERC-4337 Documentation](https://www.erc4337.io) on erc4337.io
+
+## Externally-Owned Account
+
+An externally-owned account (also known as EOA) is one of the two types of Ethereum accounts. It's controlled by a private key, it has no code and users can send messages by creating and signing Ethereum transactions.
+
+See also:
+- [Ethereum Accounts](https://ethereum.org/en/developers/docs/accounts) on ethereum.org
+- [Ethereum Whitepaper](https://ethereum.org/en/whitepaper/#ethereum-accounts) on ethereum.org
+
+## Gasless Transaction
+
+Gasless transactions (also known as meta-transactions) are Ethereum transactions that are executed by a third party called [relayer](#relayer) on behalf of a [smart account](#smart-account) to abstract the use of gas. Users need to sign a message (instead of the transaction itself) with information about the transaction they want to execute and a relayer will be in charge of creating the Ethereum transaction, signing it and executing it, paying for the gas costs. The main benefit is that users can interact with the blockchain without needing to hold the native token in their account.
```

Comment:
```suggestion
Gasless transactions (also known as meta-transactions) are Ethereum transactions that are executed by a third party called [relayer](#relayer) on behalf of a [smart account](#smart-account) to abstract the use of gas. Users must sign a message (instead of the transaction itself) with information about the transaction they want to execute. A relayer will create the Ethereum transaction, sign and execute it, and pay for the gas costs. The main benefit is that users can interact with the blockchain without holding the native token in their account.
```